src/github/graphql.ts

```diff
@@ -117,15 +120,26 @@ export interface Commit {
 	id: string;
 	commit: {
 		author: {
+			avatarUrl: string
+			name: string
+			email: string
 			user: {
 				login: string;
 				avatarUrl: string;
 				url: string;
+				email: string;
 			};
 		};
 		committer: {
-			avatarUrl: string;
-			name: string;
+			avatarUrl: string
+			name: string
+			email: string
+			user: {
+				login: string;
+				avatarUrl: string;
+				url: string;
+				email: string;
```

Comment:
Same as above: we're now getting all this info twice: once on the `commiter` property and once on the `commiter.user` property.

Reply:
See response above. However, it's worth calling out that the `committer` field appears to be unused entirely in the code base/UI.